gammapy/modeling/models/spectral.py

```diff
@@ -857,6 +858,89 @@ def evaluate(energy, index1, index2, amplitude, ebreak, reference, beta):
         return pwl * brk
 
 
+class PiecewiseBrokenPowerLawNormSpectralModel(SpectralModel):
+    """ Piecewise broken power-law spectral model
+       with renormalization at fixed energy nodes.
+       
+       For more information see :ref:`piecewise-broken-powerlaw-norm-spectral`.
+
+    Parameters
+    ----------
+    energy : `~astropy.units.Quantity`
+        Array of energies at which the model values are given (nodes).
+    norms : array
+        Array with the initial norms of the model at energies ``energy``.
+        A normalisation parameters is created for each value.
+        Default is one at each node.
+    """
+
+    tag = ["PiecewiseBrokenPowerLawNormSpectralModel", "pbpl-norm"]
+
+    def __init__(self, energy, norms=None):
+        self._energy = energy
+        if norms is None:
+            norms = np.ones(len(energy))
+        if len(norms) != len(energy):
+            raise ValueError("dimension mismatch")
+        if len(norms) < 2:
+            raise ValueError("Input arrays must contians at least 2 elements")
+        parameters = Parameters(
+            [Parameter(f"norm{k}", norm) for k, norm in enumerate(norms)]
+        )
+        self._set_default_parameters(parameters)
```

Comment:
Passing the parameters to `super().__init__()` should work and remove the need for the `_set_default_parameters`